### PR TITLE
EPSS changes

### LINKED ISSUE #198: Update section related to EPSS
Update the EPSS section within `Related Vulnerability Management Systems` based on the newly published EPSS FAQ. https://www.first.org/epss/faq

Also correct reference to Kenna to be AlientVault and Fortinet (as per data sources listed here https://www.first.org/epss/model)

### PR DESCRIPTION
**Pull request by laurie-tyz**
To close #198 

Updated the /bib file
corrected typo
updated the EPSS section.

### CODE REVIEW

File: doc/md_src_files/05_01_exploitation.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## Exploitation
 > Evidence of Active Exploitation of a Vulnerability
 
-The intent of this measure is the present state of exploitation of the vulnerability. The intent is not to predict future exploitation but only to acknowledge the current state of affairs. Predictive systems, such as EPSS, could be used to augment this decision or to notify stakeholders of likely changes [@jacobs2019exploit].
+The intent of this measure is the present state of exploitation of the vulnerability. The intent is not to predict future exploitation but only to acknowledge the current state of affairs. Predictive systems, such as EPSS, could be used to augment this decision or to notify stakeholders of likely changes [@jacobs2021epss].
```
Comment (ahouseholder) [inline doc/md_src_files/05_01_exploitation.md:4]: updated reference to published journal article

File: doc/md_src_files/12_related_systems.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # Related Vulnerability Management Systems
 
 There are several other bodies of work that are used in practice to assist vulnerability managers in making decisions.
-Three relevant systems are CVSS [@cvss_v3-1], EPSS [@jacobs2019exploit], and Tenable's Vulnerability Priority Rating ([VPR](https://www.tenable.com/blog/what-is-vpr-and-how-is-it-different-from-cvss)).
+Three relevant systems are CVSS [@cvss_v3-1], EPSS [@jacobs2021epss], and Tenable's Vulnerability Priority Rating ([VPR](https://www.tenable.com/blog/what-is-vpr-and-how-is-it-different-from-cvss)).
```
Comment (ahouseholder) [inline doc/md_src_files/12_related_systems.md:5]: updated reference to published journal article

File: doc/md_src_files/12_related_systems.md [hunk 3]
```diff
@@ -75,14 +75,14 @@ SSVC does not have such customization as a bolt-on optional metric group because
 
 ## EPSS
 
-[EPSS](https://www.first.org/epss/) is an “effort for predicting when software vulnerabilities will be exploited.”
-EPSS is currently based on a machine-learning classifier and proprietary IDS alert data from Kenna Security.
+The [Exploit Prediction Scoring System (EPSS)](https://www.first.org/epss/) is “a data-driven effort for estimating the likelihood (probability) that a software vulnerability will be exploited in the wild.”
```
Comment (ahouseholder) [inline doc/md_src_files/12_related_systems.md:78]: updated quote from FIRST web site.

File: doc/md_src_files/12_related_systems.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ In SSVC, these definitions are in the [Scope](#scope) section.
 
 The temporal metric group primarily contains the Exploit Code Maturity metric.
 This metric expresses a concept similar to [*Exploitation*](#exploitation).
-The main difference is that [*Exploitation*](#exploitation) is not optional in SSVC and that SSVC accounts for the observation that most vulnerabilities with CVE-IDs do not have public exploit code [@householder2020historical] and are not actively exploited [@guido2011exploit,@jacobs2019exploit].
+The main difference is that [*Exploitation*](#exploitation) is not optional in SSVC and that SSVC accounts for the observation that most vulnerabilities with CVE-IDs do not have public exploit code [@householder2020historical] and are not actively exploited [@guido2011exploit,@jacobs2021epss].
```
Comment (ahouseholder) [inline doc/md_src_files/12_related_systems.md:67]: updated reference to published journal article

File: doc/md_src_files/sources_ssvc.bib [hunk 5]
```diff
@@ -206,13 +206,23 @@ @inproceedings{spring2015global
...
+  volume = {2},
+  number = {3},
+  issn = {2692-1626},
+  url = {https://doi.org/10.1145/3436242},
+  doi = {10.1145/3436242},
+  journal = {Digital Threats},
+  month = {Jul},
+  articleno = {20},
+  numpages = {17},
+  keywords = {EPSS, Vulnerability management, machine learning, vulnerability exploits}
```
Comment (ahouseholder) [inline doc/md_src_files/sources_ssvc.bib:225]: updated reference to published journal article

**Review by ahouseholder — APPROVED** [review]
I made a few small mods to what @laurie-tyz had. Looks ok to me at this point.

**Comment by laurie-tyz** [comment]
Yes. Please merge.

**Pull request merged**